backend/config/jwt/middleware.py

```diff
@@ -0,0 +1,6 @@
+from graphene.types import ResolveInfo
+
+
+def allow_any(info: ResolveInfo, **kwargs):
```

Comment:
Replaces a function from `django-graphql-jwt` that simply no longer works, and seemingly is not of any importance.